Makefile

```diff
@@ -18,6 +18,7 @@ LDFLAGS=-lm
 SDKDIR = src/vendor/nrf5-sdk
 LIBDIR = src/vendor/libmira
 
+
```

Comment:
Should this be here?

Reply:
the newline or nrf5 sdk? Newline should go, nrf5-sdk needs to stay unfortunately.

Reply:
The newline